README.md

```diff
@@ -103,10 +108,6 @@ Materialize is source-available and [licensed](LICENSE) under the BSL 1.1, conve
 
 Materialize is also available as [a paid cloud service](https://materialize.com/pricing/) with additional features such as high availability via multi-active replication.
 
-## How does it work?
-
-Materialize is built on top of [differential dataflow](https://github.com/TimelyDataflow/differential-dataflow) and [timely dataflow](https://github.com/TimelyDataflow/timely-dataflow), and builds on a decade of cutting-edge stream processing research.
-
 ## For developers
 
 Materialize is written entirely in Rust.
```

Comment:
This isn’t entirely true either! It has a little bit of C.

Reply:
Changed it to "primarily written in Rust".